client/src/components/NavFooterLP.js

```diff
@@ -0,0 +1,92 @@
+import styled from 'styled-components';
+import {NavLink} from 'react-router-dom';
+
+export default function NavFooterLP() {
+  return (
+    <NavWrapper>
+      <NavLink exact to="/MapPage">
+        <NavIconMap>Start with the map.</NavIconMap>
+      </NavLink>
+      <NavLink exact to="/ListView">
+        <NavIconView>Start list view.</NavIconView>
+      </NavLink>
+      <NavLink exact to="/CreateSpotForm">
+        <NavIconSpot>Tell others your spot.</NavIconSpot>
+      </NavLink>
+      <NavLink exact to="/CreateLocationForm">
+        <NavIconLocation>Create a location.</NavIconLocation>
+      </NavLink>
+    </NavWrapper>
+  );
+}
+
+const NavWrapper = styled.footer`
+  display: grid;
+  grid-template-columns: 144px 144px;
+  grid-template-rows: 144px 144px;
+  justify-content: center;
```

Comment:
has no effect on the buttons, because the navlinks are in the way

Reply:
- background image looks nice and responsive
- logo looks nice and back home button works
- i like the idea of the circle cloud

Reply:
Thank you very much for your review and the feedback, Jens.